bundler/CHANGELOG.md

```diff
@@ -1,3 +1,58 @@
+## 2.2.0 (May xx, 2020)
+
+Highlights:
+
+  - Windows support. There's still gotchas and unimplemented features, but a Windows CI is now enforced.
+  - Full multiplatform support. Bundler should now seamlessly handle multiplatform `gems.rb` files.
```

Comment:
Should we be using `Gemfile` instead of `gems.rb`? Lots of people  I talk to about Bundler didn't know this was a thing, so I'm wondering if this may cause some confusion for users?

Reply:
This is just me trying to start raising awareness of `gems.rb` support by talking about it, I'll mention both :+1:.

Reply:
I changed it to mention both and also backfilled recent changes.